CHANGELOG.rst

```diff
@@ -2,6 +2,60 @@
+* Make `pcluster update` command safe.
+* Support DCV in China partition.
+* Use describe_instance_types to validate whether an instance type is EFA-enabled.
+* Enable support for ARM instances on Ubuntu 18.04 and Amazon Linux 2.
+* Support EFS in the China partition.
+* Enable configuration of the automatic backup features of FSx file systems.
+* Install  PMIx v3.1.5 and provide slurm support for it on all supported operating systems except for
+  CentOS 6.
+* Install glibc-static, which is required to support certain options for the Intel MPI compiler.
+
+**CHANGES**
+
+* Migrate S3 URLs to Virtual Hosted Style Access.
+* Remove hardcoded account IDs from util/generate-ami-list.py.
+* Avoid validation for ParallelCluster required additional IAM policies.
+* Removed unused dependency on supervisor from the Batch Dockerfile.
+* Move all LogGroup definitions in the CloudFormation templates into the CloudWatch substack.
+* Drop dependency on the gems ridley and ffi-libarchive.
+* Disable libvirtd service on CentOS 7. Virtual bridge interfaces are incorrectly detected by Open MPI and
+  cause MPI applications to hang, see https://www.open-mpi.org/faq/?category=tcp#tcp-selection for details 
+* Use CINC instead of Chef for provisioning instances. See https://cinc.sh/about/ for details.
+* Retry when mounting an NFS mount fails.
+* Install the `pyenv` virtual environments used by ParallelCluster cookbook and node daemon code under
+  /opt/parallelcluster instead of under /usr/local.
+* Avoid downloading the source for `env2` at installation time.
+* Vendor cookbooks as part of instance provisioning, rather than doing so before copying the cookbook into an
+  instance. Users no longer need to have `berks` installed locally.
+* Drop the dependencies on the poise-python, tar and hostname third-party cookbooks.
+* Use the new official CentOS 7 AMI as the base images for ParallelCluster AMI.
+* Upgrade NVIDIA driver to Tesla version 440.95.01 on CentOS 6 and version 450.51.05 on all other distros.
+* Upgrade CUDA library to version 11.0 on all distros besides CentOS 6.
+* Install third-party cookbook dependencies via local source, rather than using the Chef supermarket.
+* Use https wherever possible in download URLs.
+* Upgrade EFA installer to version 1.9.4:
+  * Kernel module: ``efa-1.6.0`` (from efa-1.5.1)
+  * RDMA core: ``rdma-core-28.amzn0`` (from rdma-core-25.0)
+  * Libfabric: ``libfabric-1.10.1amazon1.1`` (updated from libfabric-aws-1.9.0amzn1.1) 
+  * Open MPI: openmpi40-aws-4.0.3 (no change)
+
+**BUG FIXES**
+
+* Fix validator for CIDR-formatted IP range parameters.
+* Fix function used to convert the `extra_json` parameter to/from CloudFormation parameters.
+* Fix issue that was preventing concurrent use of custom node and pcluster CLI packages.
+* Use the correct domain name when contacting AWS services from the China partition.
+* Avoid pinning to a specific release of the Intel HPC platform.
```

Comment:
Let's be more precise here, we *are pinning* major minor version but not the precise patch release number.

Reply:
Overall, it's not even worth to mention here.